src/main/native/jgskit.mac.mak

```diff
@@ -58,23 +65,28 @@ TARGET = ${HOSTOUT}/libjgskit.dylib
 all : ${TARGET}
 
 ${TARGET} : ${OBJS}
-	gcc -shared -m64 -arch arm64 -o ${TARGET} -DMAC ${OBJS} -L ${GSKIT_HOME}/lib64 -l jgsk8iccs
+	gcc ${LDFLAGS} ${ARCHFLAGS} -o ${TARGET} ${OBJS} -L ${GSKIT_HOME}/lib64 -l jgsk8iccs
 
 ${HOSTOUT}/%.o : %.c
 	test -d ${@D} || mkdir -p ${@D} 2>/dev/null
 	gcc \
-		-fPIC \
-		${DEBUG_FLAGS} \
+		${ARCHFLAGS} \
 		${CFLAGS} \
+		${DEBUG_FLAGS} \
 		-c \
-		-I${TOPDIR}/src/main/native/ \
```

Comment:
any reason why we dont need to compile the native code under this dir?

Reply:
This is an include and it doesn't seem to be there in the linux makefile. I'm not even sure what we would need from that. Plus, it builds fine without it.

Anything you could think of that might be in there and we would need?

Reply:
This directory also I think is a duplicate of:
```
I${OPENJCEPLUS_HEADER_FILES}
```
on line 74/80 below since this is defined as :
```
OPENJCEPLUS_HEADER_FILES ?= ${TOPDIR}/src/main/native
```